### PR TITLE
Fix Issue 17952 - std.range.transposed save is invalid

### PR DESCRIPTION
**Pull request by Darredevil**
After discussing with @andralex we decided to deprecate this.
CC @schveiguy

### CODE REVIEW

**Comment by MetaLang** [comment]
The changelog explanation could be a bit longer.

**Comment by schveiguy** [comment]
Massaged the docs a bit.

BTW, I forgot to make a commit message on the first update (did it in github). Not sure how I can fix that ☹️, but at least it's on a temporary file!

**Comment by schveiguy** [comment]
Hm. The fact that there’s no doc changes worries me. Probably the docs about save (the warning in particular) should be moved to the transpose function doc.

**Comment by schveiguy** [comment]
Or... maybe just ditto the Transposed struct similar to how other functions are in the same file.

**Comment by andralex** [comment]
cc @MartinNowak

File: changelog/std-range-package-Transposed-deprecate-save.dd [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Deprecate `savce` for `std.range.package.Transposed`
```
Comment (wilzbach) [inline changelog/std-range-package-Transposed-deprecate-save.dd]: Typo: `save`

Comment (wilzbach) [reply changelog/std-range-package-Transposed-deprecate-save.dd]: Ah I didn't see the follow-up PR. Thanks @schveiguy!

**Pull request merged**